app/views/mailing_list/steps/completed/shared/_get_an_adviser_low_commitment.html.erb

```diff
@@ -1,7 +1,7 @@
-<h3 class="heading-s">Get an adviser</h3>
-
-<p>
-  Get <a href="/teacher-training-advisers"> free, one-to-one support from an adviser</a>
-  with years of teaching experience. They can help with everything from deciding if teaching
-  is right for you to how to apply. Chat by phone, text or email, as little or as often as you need.
-</p>
+<%= render CallsToAction::SimpleComponent.new(
+  title: "Understand what teaching is really like",
+  text: "Speak to our expert advisers, who have years of teaching experience. They can help with everything from deciding if teaching is right for you to how to apply. Chat by phone, text or email, as little or as often as you need.",
+  icon: "icon-person",
+  link_text: "Find out more about free advisers",
```

Comment:
This is quite long link text on mobile - wonder if it is worth shortening (removing `free` if in the heading instead?) and possibly removing `more` as not really needed?